src/community/china.md

```diff
@@ -66,10 +66,19 @@ If you’re interested in setting up your own mirror in China,
 contact [flutter-dev@googlegroups.com](mailto:flutter-dev@googlegroups.com)
 for assistance.
 
-If you're running into issues that only occur when using the `flutter-io.cn` server,
-consider reporting the issue to
+If you're running into issues that only occur when
+using the `flutter-io.cn` server, consider reporting the issue to
 [the issue tracker (镜像问题)](https://github.com/cfug/flutter.cn/issues/new/choose).
 
+## Publishing apps
```

Comment:
Taking a look at the source issue, it seems your addition covers half of the issue.

If I understood the issue correct, the other is that you can't publish to the mirror. So if a user followed the steps in [Configuring Flutter to use a mirror site](https://docs.flutter.dev/community/china#configuring-flutter-to-use-a-mirror-site) and changed your `PUB_HOSTED_URL`, publishing won't work. I guess they need to remove this override to publish.